config-model/src/main/java/com/yahoo/vespa/model/content/cluster/ContentCluster.java

```diff
@@ -591,21 +591,12 @@ public int distributionBits() {
         if (distributionBitsInPreviousModel.isPresent() && distributionBitsInPreviousModel.get() > distributionBits)
             return distributionBitsInPreviousModel.get();
 
-        if (isHosted && zone.environment() == Environment.dev) {
-            var overriddenDistributionBits = featureFlags.distributionBitsInDev();
-            if (overriddenDistributionBits == 0) return distributionBits;
-
-            log.log(INFO, "Overriding distribution bits to be " + overriddenDistributionBits);
-            return overriddenDistributionBits;
-        }
-        else {
-            return distributionBits;
-        }
+        return distributionBits;
     }
 
     private boolean zoneEnvImplies16DistributionBits() {
-        // We want perf to behave like prod as much as possible.
-        return (zone.environment() == Environment.prod) || (zone.environment() == Environment.perf);
+        // We want dev and perf to behave like prod as much as possible.
+        return Set.of(Environment.dev, Environment.perf, Environment.prod).contains(zone.environment());
```

Comment:
Looks like there's also an `isAnyOf(...)` method in `Environment`, but can't see it in use anywhere else.

Reply:
Nice! We better use it somewhere then. Fixed